funnel/models/helper.py

```diff
@@ -0,0 +1,7 @@
+reserved_words = [
+    'api', 'proposal', 'video', 'workshop', 'project', 'new', 'blog', 'post',
+    'hacknight', 'event', 'brand', 'edit', 'admin', 'section', 'venue', 'ticket',
+    'kharcha', 'funnel', 'hgtv', 'hasjob', 'boxoffice', 'app'
+]
+
+RESERVED_NAMES = reserved_words + [w + 's' for w in reserved_words]
```

Comment:
Don't do this hack. Put the explicit list of reserved names in the list, one per line (so that future amends will have a clean diff).

The file needs the standard `# -*- coding: utf-8 -*-` on top.

Reply:
This list of reserved names needs to be incorporated as well. https://github.com/hasgeek/lastuser/blob/a02560240ee40c578cb7e1992621e80681c573d9/instance/settings-sample.py#L44-L68

Reply:
Make the local list here sorted.